templates/consul_catalog.tmpl

```diff
@@ -1,12 +1,33 @@
-[backends]{{range .Nodes}}
+[backends]
+{{range .Nodes}}
     [backends.backend-{{getBackend .}}.servers.server-{{.Node.Node | replace "." "-"}}-{{.Service.Port}}]
-    url = "http://{{.Node.Address}}:{{.Service.Port}}"
+    	url = "http://{{.Node.Address}}:{{.Service.Port}}"
```

Comment:
Also, this line contains a tab, inconsistent with the spaces used in the rest of the file.